### ISSUE TITLE
GIF

### ISSUE THREAD

**Issue by grpappalardo** [grpappalardo/gee_coding_challenge_grpappalardo#1]
https://earthengine.googleapis.com/v1alpha/projects/earthengine-legacy/videoThumbnails/d112526a51ebf8762f1178cb2d54776b-145dde5604f4cfe26585299b044e1109:getPixels

**Comment by Shadrock** [comment]
Hey Gloria... this is a nice looking start, but where is the GEE gif?

**Comment by grpappalardo** [comment]
The link is updated, and it hopefully works for you.

**Comment by Shadrock** [comment]
Yep, I could see the gif this time, thanks. However, why not save it, upload it to your repo, and embed it? I believe the instructions called for this! Also, you can "close" an issues once it has been taken care of. It's common practice using Ghub to close any issues out that have been fixed.
